spec/models/ems_refresh_spec.rb

```diff
@@ -280,4 +280,16 @@ def assert_queue_item(expected_targets)
       expect(MiqQueue.count).to eq(1)
     end
   end
+
+  describe ".create_refresh_task" do
+    it "create refresh task and trancates task name to 255 symbols" do
+      ems = double("Ems")
```

Comment:
Can you create an ems with a factory? e.g. `ems = FactoryGirl.create(:ems_vmware, :name => "Ems")`

Reply:
`create_refresh_task`  called from`queue_merge`, I will move test under `queue_merger` context and will use already created instances